modules/oadp-release-notes-1-1-4.adoc

```diff
@@ -14,9 +14,27 @@ The OADP 1.1.4 release notes lists any new features, resolved issues and bugs, a
 This version of OADP is a service release. No new features are added to this version.
 
 [id="resolved-issues1.1.4_{context}"]
-== Fixed bugs
+== Resolved issues
 
-The following bugs have been fixed in this release:
+.Add support for all the velero deployment server args
+
+In previous releases of OADP, OADP did not facilitate the support of all the upstream Velero server arguments. This is issue has been resolved in OADP 1.1.4 and all the upstream Velero server arguments are supported. link:https://issues.redhat.com/browse/OADP-1557[OADP-1557]
+
+
+.Data Mover can restore from an incorrect snapshot when there was more than one VSR for the restore name and pvc name
+
+In previous releases of OADP, OADP Data Mover could restore from an incorrect snapshot if there was more than one Volume Snapshot Restore (VSR) resource in the cluster for the same Velero `restore` name and PersistentVolumeClaim (pvc) name. link:https://issues.redhat.com/browse/OADP-1822[OADP-1822]
+
+
+.Cloud Storage API BSLs need OwnerReference
+
+In previous releases of OADP, ACM BackupSchedules failed validation because of a missing `OwnerReference` on Backup Storage Locations (BSLs) created with `dpa.spec.backupLocations.bucket`. link:https://issues.redhat.com/browse/OADP-1511[OADP-1511]
+
+
+For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/browse/OADP-1557?filter=12420906[OADP 1.1.4 resolved issues] in Jira.
```

Comment:
Above link redirects to a particular bug instead of list of known issues

Reply:
Are you sure?
[check.webm](https://github.com/openshift/openshift-docs/assets/106804821/37c55793-a3cd-4ccf-82bf-d692a1f43722)

Reply:
I was able to see all the issue in list view. Thanks !